contracts/MsgOracle.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.5.12;
+
+import "node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
Please use `import "openzeppelin-solidity/contracts/ownership/Ownable.sol";` here. The import with `node_modules` doesn't play nice with coverage tooling.